fix: make doctype property's description consistent with other documents (#33509)
mdn/content: files/en-us/web/api/document/doctype/index.md

{{ApiRef("DOM")}}

> The **`doctype`** read-only property of the {{domxref("Document")}} interface returns the Document Type Declaration (DTD) associated with current document. The
> returned object implements the {{domxref("DocumentType")}} interface. Use {{domxref("DOMImplementation.createDocumentType()")}} to create a `DocumentType`.

Reviewer: ```suggestion
The **`doctype`** read-only property of the {{domxref("Document")}} interface is a {{domxref("DocumentType")}} object representing the the {{glossary("Doctype", "Document Type Declaration (DTD)")}} associated with the current document.
```

(I didn't think the second sentence was very relevant here so removed it, but please let me know if you disagree).
